A2-2015/A2-Rewrite/backend/index.js

```diff
@@ -33,32 +33,79 @@ const Product = sequelize.define('product', {
     quantity: Sequelize.INTEGER,
     price: Sequelize.DOUBLE,
     },{
-    // don't use camelcase for automatically added attributes but underscore style
-    // so updatedAt will be updated_at
     underscored: true,
-    // disable the modification of tablenames; By default, sequelize will automatically
-    // transform all passed model names (first parameter of define) into plural.
-    // if you don't want that, set the following
     freezeTableName: true,
-    // define the table's name
     tableName: 'product'
 });
 
+/*      Prooduct Category table      */
+const ProductCategory = sequelize.define('product_category', {
+    id: {
+            type: Sequelize.STRING,
+            primaryKey: true
+    },
+            created_at: Sequelize.TIME,
+            created_at: Sequelize.TIME
+    },{
+        underscored: true,
+        freezeTableName: true,
+        tableName: 'product_category'
+});
+
 const AuthLogs = sequelize.define('auth_logs', {
-    name: Sequelize.STRING,
-    email: Sequelize.STRING,
-    token: Sequelize.INTEGER,
-    event: Sequelize.INTEGER,
-},{
-    // don't use camelcase for automatically added attributes but underscore style
-    // so updatedAt will be updated_at
-    underscored: true,
-    // disable the modification of tablenames; By default, sequelize will automatically
-    // transform all passed model names (first parameter of define) into plural.
-    // if you don't want that, set the following
-    freezeTableName: true,
-    // define the table's name
-    tableName: 'auth_logs'
+        name: Sequelize.STRING,
+        email: Sequelize.STRING,
+        token: Sequelize.INTEGER,
+        event: Sequelize.INTEGER,
+    },{
+        underscored: true,
+        freezeTableName: true,
+        tableName: 'auth_logs'
+});
+
+/*      Order table      */
+const Orders = sequelize.define('order', {
+        id: {
+            type: Sequelize.INTEGER,
+            primaryKey: true
+        },
+        created_at: Sequelize.TIME,
+        customer_first_name: Sequelize.STRING,
+        customer_last_name: Sequelize.STRING,
+        customer_address: Sequelize.STRING,
+        customer_phone: Sequelize.STRING,
+        price: Sequelize.DOUBLE,
+        shipped_flag: Sequelize.BOOLEAN
+    },{
+        underscored: true,
+        freezeTableName: true,
+        tableName: 'order',
+        timestamps: false
```

Comment:
Why not leveraging sequelize timestamps?

Reply:
we can.. I just didn't put them in the schema originally as I don't know how they would be used.. should I add?